.github/workflows/release-new-version.yml

```
@@ -74,7 +74,17 @@ jobs:
             connector=$(basename ${connector_dir})
             newman dir-import ${POSTMAN_DIR}/${connector} -o ${POSTMAN_JSON_DIR}/${connector}.postman_collection.json
           done
-          (git diff --quiet && git diff --staged --quiet) || (git commit -am 'test(postman): update postman collection files' && echo "Committed changes") || (echo "Unable to commit the following changes:" && git diff)
+
+          if git diff --quiet && git diff --staged --quiet; then
+              echo "No changes detected."
+          elif git add postman; then
+              echo "Changes have been staged."
+          elif git commit -am 'test(postman): update postman collection files'; then
+              echo "Committed changes."
+          else
+              echo "Unable to commit the following changes:"
+              git diff
+          fi
```

Can't we just do something like:

```suggestion
          git add postman
          if ! git diff --staged --quiet postman; then
            git commit --message 'test(postman): update postman collection files'
          fi
```

Performing the `git add` before checking for changed files also handles untracked files in the `postman` directory.
And any errors in the workflow should stop the workflow, which I think is fine.

---

Had a similar thought.